### PR TITLE
Stack using arrays java

### PR DESCRIPTION
**Pull request by khusheekapoor**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes, I have read the Contributing Guidelines on Pull Requests

### Description

I have written a code to implement basic operations (pushing, popping and displaying) on Stacks using Arrays in Java as a GSSoC'21 participant.

### Checklist

- [Yes] I've read the contribution guidelines.
- [Yes] I've checked the issue list before deciding what to submit.
- [Yes] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

For issue #5352

### CODE REVIEW

File: Java/ds/StackUsingArrays.java [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+        System.out.println("Enter size of stack: ");
+        int size = scanner.nextInt();
+        scanner.nextLine();
+
+        StackUsingArrays ob = new StackUsingArrays(size);
+
+        System.out.println("Choose from the following: ");
+        System.out.println("1. Insert an element");
+        System.out.println("2. Delete an element");
+        System.out.println("3. Diplaying the stack");
```
Comment (ankitaggarwal23) [inline Java/ds/StackUsingArrays.java]: Fix typo of displaying

File: Java/ds/StackUsingArrays.java [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+            scanner.nextLine();
+
+            switch(choice)
+            {
+                case 1:
+                        System.out.println("Enter value: ");
+                        int value = scanner.nextInt();
+                        scanner.nextLine();
+                            ob.push(value);
+                        break;
```
Comment (ankitaggarwal23) [inline Java/ds/StackUsingArrays.java]: Align it

File: Java/ds/StackUsingArrays.java [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+                case 2:
+                            ob.pop();
+                        break;
```
Comment (ankitaggarwal23) [inline Java/ds/StackUsingArrays.java]: Align it

File: Java/ds/StackUsingArrays.java [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+
+                case 3:
+                            ob.display();
+                        break;
```
Comment (ankitaggarwal23) [inline Java/ds/StackUsingArrays.java]: Align it

File: Java/ds/StackUsingArrays.java [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+
+/*
+ * Test Case-
+ * 
+Enter size of stack: 
+3
+Choose from the following: 
+1. Insert an element
+2. Delete an element
+3. Diplaying the stack
```
Comment (ankitaggarwal23) [inline Java/ds/StackUsingArrays.java]: Fix typo of displaying

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: Java/ds/StackUsingArrays.java (hunk 1 above)
Comment (khusheekapoor) [reply Java/ds/StackUsingArrays.java]: Changes made.

File: Java/ds/StackUsingArrays.java (hunk 2 above)
Comment (khusheekapoor) [reply Java/ds/StackUsingArrays.java]: Changes made.

File: Java/ds/StackUsingArrays.java (hunk 3 above)
Comment (khusheekapoor) [reply Java/ds/StackUsingArrays.java]: Changes made.

File: Java/ds/StackUsingArrays.java (hunk 4 above)
Comment (khusheekapoor) [reply Java/ds/StackUsingArrays.java]: Changes made.

File: Java/ds/StackUsingArrays.java (hunk 5 above)
Comment (khusheekapoor) [reply Java/ds/StackUsingArrays.java]: Changes made.

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**